Assets/HoloToolkit/Input/Scripts/Interactions/TapToPlace.cs

```diff
@@ -38,6 +38,9 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         [Tooltip("Setting this to true will enable the user to move and place the object in the scene without needing to tap on the object. Useful when you want to place an object immediately.")]
         public bool IsBeingPlaced;
 
+        [Tooltip("Setting this to true will this behavior control the DrawMesh property on the spatial mapping")]
+        public bool ControlDrawMeshOnSpatialMappingWhenBeeingPlaced = true;
```

Comment:
Also, any way to shorten this bool name? I don't think the whole thing will show up in the Inspector. Maybe "DrawMeshesDuringPlacement" or "AllowMeshVisualizationControl" (which may still be too long).